webassembly/F3DEmscriptenBindings.cxx

```diff
@@ -28,8 +28,8 @@ f3d::options* getOptionsPtr(f3d::engine& e) { return &e.getOptions(); }
 f3d::options* toggle(f3d::options& o, const std::string& name) { return &o.toggle(name); }
 
 f3d::loader* getLoaderPtr(f3d::engine& e) { return &e.getLoader(); }
-f3d::loader* addFile(f3d::loader& l, const std::string& p) { return &l.addFile(p); }
-void loadFile(f3d::loader& l) { l.loadFile(); }
+f3d::loader* loadGeometry(f3d::loader& l, const std::string& p) { return &l.loadGeometry(p); }
```

Comment:
You have to change the HTML example file too. Go ahead blindly it should be straigthforward, I'll double check it still work later.

Reply:
on it

Reply:
I don't see the change to the HTML file

Reply:
done this time